core/src/main/java/cucumber/runtime/xstream/LocalizedXStreams.java

```diff
@@ -37,6 +48,21 @@ private LocalizedXStream newXStream(Locale locale) {
         return new LocalizedXStream(classLoader, lookup, lookup, locale);
     }
 
+    private void registerExtraConverters(LocalizedXStream xStream) {
+        for (XStreamConverter converter : extraConverters) {
+            try {
+                ConverterMatcher matcher = converter.value().newInstance();
+                if (matcher instanceof Converter) {
+                    xStream.registerConverter((Converter) matcher, converter.priority());
+                }
```

Comment:
I just came across this in attempting to use this feature (which is excellent!) It looks like `SingleValueConverter`s are treated separately from other `Converters` and are not type-compatible, so in order to properly use `SingleValueConverter`s with the global registration, there would need to be a second if statement here to handle them.